automatic/julia/tools/chocolateyinstall.ps1

```diff
@@ -10,7 +10,7 @@ $packageArgs = @{
 
   softwareName  = 'Julia*'
 
-  silentArgs    = '/S'
+  silentArgs    = '/VERYSILENT'
```

Comment:
I assume their installer is now an `InnSetup` installer, if so can you please make the following change:
```suggestion
  silentArgs    = '/VERYSILENT /SUPPRESSMSGBOXES /NORESTART /SP-'
```